frontend/app/craco.config.js

```diff
@@ -23,6 +23,12 @@ module.exports = {
   },
   jest: {
     configure: jestConfig => {
+      const path = require("path")
+      // use local files for @streamlit/lib
+      jestConfig.moduleNameMapper["^@streamlit/lib$"] = path.resolve(
```

Comment:
Why do we need this? If we remove this, would the path resolving into work anymore?

Reply:
Oki, maybe you can add this info to the comment that his is needed for `yarn test`.